lms/envs/test.py

```diff
@@ -569,33 +569,23 @@
 COMPREHENSIVE_THEME_DIRS = [REPO_ROOT / "themes", REPO_ROOT / "common/test"]
 COMPREHENSIVE_THEME_LOCALE_PATHS = [REPO_ROOT / "themes/conf/locale", ]
 
-SERVICE_VARIANT = os.environ.get('SERVICE_VARIANT', None)
-
-CONFIG_ROOT = path(os.environ.get('CONFIG_ROOT', ENV_ROOT))
-
-CONFIG_PREFIX = SERVICE_VARIANT + "." if SERVICE_VARIANT else ""
-
-with open(CONFIG_ROOT / CONFIG_PREFIX + "auth.json") as auth_file:
-    AUTH_TOKENS = json.load(auth_file)
-
+##################### PhilU Settings #######################
 LMS_ROOT_URL = "http://local.philanthropyu.org:8000"
 NODEBB_RETRY_DELAY = 60
 NODEBB_ENDPOINT = "http://local.philanthropyu.org:4567"
 # replace NODEBB_MASTER_TOKEN with value from your setup
-NODEBB_MASTER_TOKEN = AUTH_TOKENS.get("NODEBB_MASTER_TOKEN")
-MANDRILL_API_KEY = AUTH_TOKENS.get("MANDRILL_API_KEY")
-MAILCHIMP_API_KEY = AUTH_TOKENS.get("MAILCHIMP_API_KEY")
-MAILCHIMP_LEARNERS_LIST_ID = ""
-
-with open(CONFIG_ROOT / CONFIG_PREFIX + "env.json") as env_file:
-    ENV_TOKENS = json.load(env_file)
-
-if ENV_TOKENS.get('COMPREHENSIVE_THEME_DIR'):
-    COMPREHENSIVE_THEME_DIR = ENV_TOKENS.get('COMPREHENSIVE_THEME_DIR')
-
-COMPREHENSIVE_THEME_DIRS = ENV_TOKENS.get('COMPREHENSIVE_THEME_DIRS', COMPREHENSIVE_THEME_DIRS) or []
-COMPREHENSIVE_THEME_LOCALE_PATHS = ENV_TOKENS.get('COMPREHENSIVE_THEME_LOCALE_PATHS', [])
+NODEBB_MASTER_TOKEN = 'test-master-token-nodebb'
+MANDRILL_API_KEY = 'test_mandrill_api_key_part1-part2-part3'
+# TODO: Mock usages of mailchimp client in all unit tests and replace this with dummy key
+MAILCHIMP_API_KEY = '21bf1dc52e6ef64a6a3f573e4ea9a999-us12'
```

Comment:
If this is an actual key, then we should not keep it in code. Even it should not be in git history

Reply:
This is real and is being using for development and our local environments. The extent to which I researched upon this, I didn't find any alternative so I've added a TODO to mock the usages of MAIL_CHIMP client.
One alternative was to use it from environment variable, but you our testing server is not a dedicated server so we can export anything manually there. Any environment variable we have to export, we export it in `scripts/Jenkinsfiles/python-unittest` which is also a public file, so only solution is to mock the usages, which was too much for the scope of this ticket, so I've added a TODO.

Reply:
@ahmed-faraz46 Can't we keep it in environment files something like `test.env.json` ??

Reply:
Nope, in our testing server there no environment files. The whole setup is launched from the code and a worker is created when a PR is made. There is no existing server. But I just found a solution to this problem, we will move MAILCHIMP_API_KEY in credentials repo (devops is managing this repo for philu) and we are already fetching a few thing from that repo, we will fetch this key and will export in environment programmatically and use it from environment here. @zeshanarifios @zahid-arbisoft